roles/test/visualmesh.role

```diff
@@ -14,8 +14,7 @@ nuclear_role(
   support::configuration::SoccerConfig
   # Sensors
   input::Camera
-  platform::cm740::HardwareIO
-  # platform::cm740::HardwareSimulator
+  platform::<subcontroller>::HardwareIO
```

Comment:
Yea, this seems like a horrible way to do this

Reply:
These are cmake files, just use cmake variables like `${NUBOTS_SUBCONTROLLER}`